vuln/npm/500.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": 500,
+  "title": "Buffer Over-read",
+  "overview": "A specifically malformed MQTT Subscribe packet crashes MQTT Brokers using the mqtt-packet module for decoding  ",
```

Comment:
Should the overview be correctly puncutated? It has trailing whitespace and no period (but maybe that doesn't matter).

Reply:
Not really since we're automatically just pasting it from the hackerone report

Reply:
OK